draft-ietf-quic-transport.md

```diff
@@ -4382,17 +4382,10 @@ the difference between the largest acknowledged packet and packet number being
 sent.  A peer receiving the packet will then correctly decode the packet number,
 unless the packet is delayed in transit such that it arrives after many
 higher-numbered packets have been received.  An endpoint SHOULD use a large
-enough packet number encoding to allow the packet number to be recovered even
-if the packet arrives after packets that are sent afterwards.
-
-As a result, the size of the packet number encoding is at least one bit more
-than the base-2 logarithm of the number of contiguous unacknowledged packet
-numbers, including the new packet.
```

Comment:
I think my preferences goes to retaining this sentence.

Having pseudo code makes sense, but IMO it is better to state the constraints that a function has to meet.

Reply:
It's not actually gone, it's a comment in the pseudo-code.  The paragraph above describes the constraints, while this describes the math which would satisfy those complaints.

Reply:
I think my point is that it would be more helpful to the readers to have this paragraph here, than just having the super-abstract guidance stated in the paragraph above. I tend to think that the specification should be parsable without reading the pseudo-code.

That said, this is a discussion about where the practical guidance should be stated, and therefore my preference is not a strong at all.